velox/connectors/hive/storage_adapters/s3fs/tests/S3FileSystemTest.cpp

```diff
@@ -340,4 +340,33 @@ TEST_F(S3FileSystemTest, registerCredentialProviderFactories) {
           }),
       "CredentialsProviderFactory 'my-credentials-provider' already registered");
 }
+
+TEST_F(S3FileSystemTest, list) {
```

Comment:
Let's test this along with the existing `writeFileAndRead` test. No need for another test.